th2_data_services/interfaces/events_tree/events_tree_collection.py

```diff
@@ -68,9 +86,17 @@ def get_parentless_trees(self) -> List[EventsTree]:
             List of parentless trees if they exist, otherwise empty list.
         """
         if self._parentless is not None:
+            w = "This instance of ETC have a list of parentless trees by detached events."
+            self._logger.warning(w)
+            warnings.warn(w)
+            self._warnings.append(w)
             return self._parentless
         else:
             self._parentless = self._build_parentless_trees()
+            w = "You have created a list of parentless trees by detached events."
+            self._logger.warning(w)
+            warnings.warn(w)
+            self._warnings.append(w)
```

Comment:
Why do we need these warnings? What do we want to say user?

Reply:
My opinion - we don't need these messages.

Reply:
Removed from events_tree_collection.py https://github.com/th2-net/th2-data-services/commit/0460ed688d47f24a298ed465e68c81b9cfbf7118